carsondrobe/Fellas
# Twilio setup (registration welcome text)

branched from #170 

Added integration for sign up welcome text. 

Steps to test:

 **DONE already in class for everyone should be good to just start at step 1.** Add your phone number to [the Twilio console](https://console.twilio.com/?frameUrl=/console/phone-numbers/verified) you need to verify your phone number on the Twilio trial it won't send to an "unverified number". 
Note: you may it may text me a 2FA code when you login to try to login to twilio, it needed a phone number on the account. I'll just text the code to the groupchat if I get a 2fa code text 

3. Add the required dependencies from `requirements.txt` the new ones are `Twilio` and `python-dotenv`

4. Create a file named `.env` in the root directory of the project (same level as the git ignore)

5. add the credentials for the twillio number and identity keys provided in discord.

2 test run examples:

![IMG_278CA0C7D6E0-1](https://github.com/carsondrobe/Fellas/assets/112982183/7261203e-d3f4-497c-acff-269877b06565)
